### ISSUE TITLE
can't reproduce the r50

### ISSUE THREAD

**Issue by loserhou** [youngskkim/CRN#19]
I use 4 a800 and bs＝8
mAP: 0.4681
mATE: 0.5202
mASE: 0.2827
mAOE: 0.5189
mAVE: 0.2834
mAAE: 0.1873
NDS: 0.5548
Eval time: 111.6s 
but your map is 47.3 and nds is 56.2,can you give me some advice?

**Comment by adeelajmal2468** [comment]
@loserhou 
![image](https://github.com/youngskkim/CRN/assets/51111457/c1c08efe-dd72-4d84-9dcf-50920c6941a8)
results on CRN with r50

**Comment by loserhou** [comment]
> @loserhou 
> ![image](https://github.com/youngskkim/CRN/assets/51111457/c1c08efe-dd72-4d84-9dcf-50920c6941a8)
> These are my results on CRN with r50
how to set the learning rate?

**Comment by loserhou** [comment]
> > @loserhou 
> > ![image](https://github.com/youngskkim/CRN/assets/51111457/c1c08efe-dd72-4d84-9dcf-50920c6941a8)
> > These are my results on CRN with r50
> how to set the learning rate?

the original lr/8？

**Comment by loserhou** [comment]
> @loserhou 
> ![image](https://github.com/youngskkim/CRN/assets/51111457/c1c08efe-dd72-4d84-9dcf-50920c6941a8)
> These are my results on CRN with r50

is this the weights given by the author or did yiu train them yourself?

**Comment by adeelajmal2468** [comment]
The checkpoints were given

**Comment by loserhou** [comment]
> The checkpoints were given

have you trained by yourself?

**Comment by adeelajmal2468** [comment]
Yes I did try it but it was taking too long for 1 epoch. So for now I just tested it

**Comment by loserhou** [comment]
> Yes I did try it but it was taking too long for 1 epoch. So for now I just tested it 

thx

**Comment by youngskkim** [comment]
Different seeds or GPU architectures may result in performance differences of up to a few percent.

**Comment by L1bertad** [comment]
@loserhou May I ask if you have obtained results similar to those in the paper?
